packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeComponent.tsx

```diff
@@ -127,8 +127,11 @@ function ModelsTreeComponentImpl(props: ModelTreeComponentProps & { iModel: IMod
   const queryModels = useCallback(async (): Promise<ModelInfo[]> => {
     const queryParams: ModelQueryParams = {
       from: "BisCore.GeometricModel3d",
+      where: `GeometricModel3d.ModeledElement.Id IN (select ECInstanceId FROM BisCore.GeometricElement3d)
+      OR GeometricModel3d.ModeledElement.Id IN (select ECInstanceId FROM BisORe.InformationPartitionElement)`,
       wantPrivate: false,
     };
+
     const modelProps = await iModel.models.queryProps(queryParams);
```

Comment:
I would suggest moving models querying to separate function. This will make component code cleaner and simplifies testing as there will be no need to stub UI stuff.